driver/csiplugin/server.go

```diff
@@ -97,6 +98,10 @@ func (s *nonBlockingGRPCServer) serve(endpoint string, ids csi.IdentityServer, c
 	if err != nil {
 		klog.Fatalf("Failed to listen: %v", err)
 	}
+	// Change csi.sock file permsion to read and write only
```

Comment:
please address the typo , permsion -> permission

Reply:
Thanks, have updated the comment.